api/drf_views.py

```diff
@@ -143,6 +145,8 @@ def get_object(self):
             )
 
     def get_serializer_class(self):
+        if self.request.GET.get('mini', None) == 'true':
```

Comment:
@GregoryHorvath You may want to use `lower` so that both `true`/`True` have the same meaning.
```suggestion
        if self.request.GET.get('mini', 'false').lower() == 'true':
```